core/workflow/loadDPL.go

```diff
@@ -57,5 +62,95 @@ func LoadDPL(tasks []*task.Class) (workflow Role, err error) {
 	}
 
 	workflow = root
```

Comment:
How should we decide a `name` for the `root` role? The DPL is only information about the constituent tasks. Possible choices:
- use the name of the executable
- executable name along with some logic to derive a `workflow` name based on the constituent tasks

Reply:
I don't think inferring a name from the executables would be reliable or useful.

What about using the filename of the input DPL dump?

Reply:
Is there a pattern for the input DPL Dumps? From my experience with O2, it’s the user who provides the name for the DPL Dump.

I’m not against this idea. But this could potentially end up with workflow names being irrelevant to the content.